docs/build/apps/dapp-frontend.mdx

```diff
@@ -1,561 +1,536 @@
+  smartContracts.push({
+    alias: filenameNoExtension(wasm),
+    wasm: wasm,
+    contractid: stdout.trimEnd(),
+  });
 
-Right now, the wallet that best supports Soroban is [Freighter](../guides/freighter/README.mdx). It is available as a Firefox Add-on, as well as extensions for Chrome and Brave. Go ahead and [install it now](https://freighter.app).
+  console.log(`Deployed ${filenameNoExtension(wasm)}`);
+}
+
+async function deployAll() {
+  console.log("Deploying all contracts");
+  const wasmFiles = glob(`${dirname}/target/wasm32v1-none/release/*.wasm`);
 
-Once it's installed, open it up by clicking the extension icon. If this is your first time using Freighter, you will need to create a new wallet. Go through the prompts to create a password and save your recovery passphrase.
+  for (const wasm of wasmFiles) {
+    await deploy(wasm);
+  }
+}
+```
 
-Go to Settings (the gear icon) → Preferences and toggle the switch to Enable Experimental Mode. Then go back to its home screen and select "Test Net" from the top-right dropdown. Finally, if it shows the message that your Stellar address is not funded, go ahead and click the "Fund with Friendbot" button.
+The `deploy()` function will get the contract ID from the CLI call, and add the contract name, wasm file path, and contract ID to the `smartContracts[]` array.
 
-Now you're all set up to use Freighter as a user, and you can add it to your app.
+In this example, we only use one smart contract, but it’s not uncommon to use multiple smart contracts in a dapp, so the template supports the use of multiple contracts.
 
-### Add the StellarWalletsKit and set it up
+##### Create bindings
 
-Even though we're using Freighter to test our app, there are more wallets that support signing smart contract transactions. To make their integration easier, we are using the `StellarWalletsKit` library which allows us support all Stellar Wallets with a single library.
+The Stellar CLI has a convenient command to create an NPM package that makes it easy to call smart contract functions from a JavaScript/TypeScript-based frontend. We call the package “bindings” because that’s what it does: it binds the contract and the frontend together.
 
-To install this kit we are going to include the next package:
+As with the contract build functions, the binding function is also capable of handling multiple contracts, so there’s a function for creating the binding package for a contract (`bind()`) and a function that calls `bind()` for each contract (`bindAll()`).
 
-```shell
-npm install @creit.tech/stellar-wallets-kit
+```javascript
+function bind({ alias, wasm, contractid }) {
+  // Create bindings for a deployed contract
+  execSync(
+    `stellar contract bindings typescript --contract-id ${contractid} --output-dir ${dirname}/packages/${alias} --overwrite`,
+  );
+  // Build the package
+  execSync(`(cd ${dirname}/packages/${alias} && npm i && npm run build)`);
```

Comment:
might wanna add `&& cd ${dirname}` or something similar to the end of this command?